hermes/probe/read/read.go

```diff
@@ -0,0 +1,124 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Author: Alicja Kwiecinska, GitHub: alicjakwie
+//
+// package read contains all of the logic necessary to verify the availability and consistency of the file contents and names in GCS.
+//
+// TODO(#76) change the type of fileID to int
+package read
+
+import (
+	"context"
+	"crypto/sha1"
+	"fmt"
+	"io"
+	"time"
+
+	"cloud.google.com/go/storage"
+	"github.com/google/cloudprober/logger"
+	"github.com/googleapis/google-cloud-go-testing/storage/stiface"
+	"github.com/googleinterns/step224-2020/hermes/probe"
+	"github.com/googleinterns/step224-2020/hermes/probe/metrics"
+	"google.golang.org/api/iterator"
+)
+
+const (
+	FileNameFormat          = "Hermes_%02d_%x"
+	minFileID               = 1
+	maxFileID               = 50
+	maxFileSizeBytes        = 1000
+	hermesAPILatencySeconds = "hermes_api_latency_s"
```

Comment:
Isn't this in the `metrics` package now?

Reply:
It is but not as a constant as in (changing this now it's) "hermes_api_latency_seconds"

Reply:
left it as a constant will be removed once the metrics.go PR is merged